can't build with ddnet-libs cloned

I just set up arch linux with my new pc and can't build ddnet, even though I cloned with ddnet-libs.
I didn't have vulkan installed.
This is the error:
```
[lukas@medion build]$ cmake ../
-- The C compiler identification is GNU 12.1.0
-- The CXX compiler identification is GNU 12.1.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: /usr/bin/cc - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Detecting CXX compiler ABI info
-- Detecting CXX compiler ABI info - done
-- Check for working CXX compiler: /usr/bin/c++ - skipped
-- Detecting CXX compile features
-- Detecting CXX compile features - done
-- Performing Test HAVE_ATOMICS_WITHOUT_LIB
-- Performing Test HAVE_ATOMICS_WITHOUT_LIB - Success
-- Looking for __i386
-- Looking for __i386 - not found
-- Performing Test FLAG_SUPPORTED_fuse_ld_mold
-- Performing Test FLAG_SUPPORTED_fuse_ld_mold - Failed
-- Performing Test FLAG_SUPPORTED_fuse_ld_lld
-- Performing Test FLAG_SUPPORTED_fuse_ld_lld - Failed
-- Performing Test FLAG_SUPPORTED_fuse_ld_gold
-- Performing Test FLAG_SUPPORTED_fuse_ld_gold - Success
-- Using gold linker
-- Performing Test FLAG_SUPPORTED_fstack_protector_all
-- Performing Test FLAG_SUPPORTED_fstack_protector_all - Success
-- Performing Test FLAG_SUPPORTED_fno_exceptions
-- Performing Test FLAG_SUPPORTED_fno_exceptions - Success
-- Performing Test FLAG_SUPPORTED_Wall
-- Performing Test FLAG_SUPPORTED_Wall - Success
-- Performing Test FLAG_SUPPORTED_Wdeclaration_after_statement
-- Performing Test FLAG_SUPPORTED_Wdeclaration_after_statement - Failed
-- Performing Test FLAG_SUPPORTED_Wextra
-- Performing Test FLAG_SUPPORTED_Wextra - Success
-- Performing Test FLAG_SUPPORTED_Wno_unused_parameter
-- Performing Test FLAG_SUPPORTED_Wno_unused_parameter - Success
-- Performing Test FLAG_SUPPORTED_Wno_missing_field_initializers
-- Performing Test FLAG_SUPPORTED_Wno_missing_field_initializers - Success
-- Performing Test FLAG_SUPPORTED_Wformat_2
-- Performing Test FLAG_SUPPORTED_Wformat_2 - Success
-- Performing Test FLAG_SUPPORTED_Wno_implicit_function_declaration
-- Performing Test FLAG_SUPPORTED_Wno_implicit_function_declaration - Success
-- Performing Test FLAG_SUPPORTED_Wno_nullability_completeness
-- Performing Test FLAG_SUPPORTED_Wno_nullability_completeness - Success
-- Performing Test FLAG_SUPPORTED_Wduplicated_cond
-- Performing Test FLAG_SUPPORTED_Wduplicated_cond - Success
-- Performing Test FLAG_SUPPORTED_Wduplicated_branches
-- Performing Test FLAG_SUPPORTED_Wduplicated_branches - Success
-- Performing Test FLAG_SUPPORTED_Wlogical_op
-- Performing Test FLAG_SUPPORTED_Wlogical_op - Success
-- Performing Test FLAG_SUPPORTED_Wrestrict
-- Performing Test FLAG_SUPPORTED_Wrestrict - Success
-- Performing Test FLAG_SUPPORTED_Wshadow_all
-- Performing Test FLAG_SUPPORTED_Wshadow_all - Failed
-- Performing Test FLAG_SUPPORTED_Wshadow_global
-- Performing Test FLAG_SUPPORTED_Wshadow_global - Success
-- Performing Test FLAG_SUPPORTED_Wthread_safety
-- Performing Test FLAG_SUPPORTED_Wthread_safety - Failed
-- Performing Test FLAG_SUPPORTED_Wthread_safety_negative
-- Performing Test FLAG_SUPPORTED_Wthread_safety_negative - Failed
-- Performing Test FLAG_SUPPORTED_Wsuggest_override
-- Performing Test FLAG_SUPPORTED_Wsuggest_override - Success
-- Performing Test FLAG_SUPPORTED_Wno_alloc_size_larger_than
-- Performing Test FLAG_SUPPORTED_Wno_alloc_size_larger_than - Success
-- Performing Test DEFINE_FORTIFY_SOURCE
-- Performing Test DEFINE_FORTIFY_SOURCE - Success
-- Found PkgConfig: /usr/bin/pkg-config (found version "1.8.0")
-- Found ZLIB: /usr/lib/libz.so (found version "1.2.12")
-- Found OpenSSL: /usr/lib/libcrypto.so (found version "1.1.1o")
-- Found Crypto: /usr/lib/libcrypto.so
-- Checking for module 'libcurl'
--   Found libcurl, version 7.83.1
-- Found Curl: /usr/lib/libcurl.so
-- Checking for module 'libavcodec'
--   Found libavcodec, version 59.18.100
-- Checking for module 'libavformat'
--   Found libavformat, version 59.16.100
-- Checking for module 'libavutil'
--   Found libavutil, version 57.17.100
-- Checking for module 'libswscale'
--   Found libswscale, version 6.4.100
-- Checking for module 'libswresample'
--   Found libswresample, version 4.3.100
-- Checking for one of the modules 'libx264;x264'
-- Found FFMPEG: /usr/lib/libavcodec.so
-- Checking for module 'freetype2'
--   Found freetype2, version 24.3.18
-- Found Freetype: /usr/lib/libfreetype.so
-- Found GLEW: /usr/include (found version "2.2.0")
-- Found GTest: /usr/lib64/cmake/GTest/GTestConfig.cmake (found version "1.11.0")
-- Checking for module 'ogg'
--   Found ogg, version 1.3.5
-- Found Ogg: /usr/include
-- Checking for module 'opus'
--   Found opus, version 1.3.1
-- Found Opus: /usr/include/opus
-- Checking for module 'opusfile'
--   Package 'opusfile', required by 'virtual:world', not found
-- Found Opusfile: /home/lukas/ddnet/ddnet-libs/opus/linux/lib64/libopusfile.a
-- Checking for module 'pnglite'
--   Package 'pnglite', required by 'virtual:world', not found
-- Found Pnglite: src/engine/external/pnglite
-- Found PythonInterp: /usr/bin/python3 (found suitable version "3.10.5", minimum required is "3")
-- Checking for module 'sdl2'
--   Found sdl2, version 2.0.22
-- Found SDL2: /usr/lib/libSDL2.so
-- Found SQLite3: /usr/include (found version "3.38.5")
-- Checking for module 'sqlite3'
--   Found sqlite3, version 3.38.5
-- Found SQLite3: /usr/include
-- Found Threads: TRUE
-- Checking for module 'wavpack'
--   Found wavpack, version 5.4.0
-- Found Wavpack: /usr/lib/libwavpack.so
-- Checking for module 'vulkan'
--   Found vulkan, version 1.3.213
-- Building vulkan shaders
/home/lukas/ddnet/data/shader/vulkan/quad.vert
-- Finished building vulkan shaders
-- ******** DDNet ********
-- Target OS: linux x86_64
-- Compiler: /usr/bin/c++
-- Build type: Release
-- Dependencies:
--  * Curl found
--  * FFmpeg found
--  * Freetype found
--  * Glew found
--  * GTest found
--  * Ogg found
--  * OpenSSL Crypto found
--  * Opus found
--  * Opusfile not found (using bundled version)
--  * Pnglite not found (using bundled version)
--  * PythonInterp found
--  * SDL2 found
--  * SQLite3 found
--  * Wavpack found
--  * Zlib found
--  * Vulkan found
-- Found Notify: notify;gdk_pixbuf-2.0;gio-2.0;gobject-2.0;glib-2.0
-- Found OpenGL: /usr/lib/libOpenGL.so
-- Looking for WavpackOpenFileInputEx
-- Looking for WavpackOpenFileInputEx - found
-- Looking for WavpackCloseFile
-- Looking for WavpackCloseFile - found
-- Configuring done
-- Generating done
-- Build files have been written to: /home/lukas/ddnet/build
[lukas@medion build]$ make -j $(nproc)
[  2%] Generating src/game/generated/protocol.h
[  2%] Generating src/game/generated/protocol7.h
[  2%] Generating src/game/generated/git_revision.cpp
[  2%] Generating src/game/generated/protocolglue.h
[  3%] Generating src/game/generated/protocolglue.h
[  3%] Building CXX object CMakeFiles/steam_api.dir/src/steam/steam_api_stub.cpp.o
[  3%] Building C object CMakeFiles/json.dir/src/engine/external/json-parser/json.c.o
[  4%] Building C object CMakeFiles/pnglite.dir/src/engine/external/pnglite/pnglite.c.o
[  6%] Generating src/game/generated/protocol7.cpp
[  6%] Generating src/game/generated/protocol.cpp
[  6%] Generating src/game/generated/protocolglue.cpp
[  6%] Building CXX object CMakeFiles/game-shared.dir/src/game/collision.cpp.o
[  6%] Building CXX object CMakeFiles/game-shared.dir/src/game/bezier.cpp.o
[  7%] Building CXX object CMakeFiles/game-shared.dir/src/game/gamecore.cpp.o
[  7%] Building CXX object CMakeFiles/game-shared.dir/src/game/layers.cpp.o
[  8%] Linking CXX static library libsteam_api.a
[  8%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/assertion_logger.cpp.o
[  8%] Building CXX object CMakeFiles/game-shared.dir/src/game/localization.cpp.o
[  8%] Built target steam_api
[  9%] Building CXX object CMakeFiles/game-shared.dir/src/game/mapbugs.cpp.o
[  9%] Building CXX object CMakeFiles/game-shared.dir/src/game/mapitems.cpp.o
[  9%] Building CXX object CMakeFiles/game-shared.dir/src/game/prng.cpp.o
[ 10%] Building CXX object CMakeFiles/game-shared.dir/src/game/teamscore.cpp.o
[ 10%] Built target json
[ 10%] Building CXX object CMakeFiles/game-shared.dir/src/game/generated/git_revision.cpp.o
[ 12%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/compression.cpp.o
[ 12%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/config.cpp.o
[ 12%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/console.cpp.o
[ 12%] Built target pnglite
[ 13%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/csv.cpp.o
[ 13%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/datafile.cpp.o
[ 13%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/demo.cpp.o
[ 14%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/econ.cpp.o
[ 14%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/engine.cpp.o
[ 14%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/fifo.cpp.o
[ 15%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/global_uuid_manager.cpp.o
[ 15%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/filecollection.cpp.o
[ 15%] Built target game-shared
[ 15%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/http.cpp.o
[ 16%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/huffman.cpp.o
[ 16%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/image_manipulation.cpp.o
[ 16%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/jobs.cpp.o
[ 18%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/json.cpp.o
[ 18%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/kernel.cpp.o
[ 18%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/linereader.cpp.o
[ 19%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/map.cpp.o
[ 19%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/masterserver.cpp.o
[ 19%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/memheap.cpp.o
[ 20%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/netban.cpp.o
[ 20%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/network.cpp.o
[ 20%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/network_client.cpp.o
[ 21%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/network_conn.cpp.o
[ 21%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/network_console.cpp.o
[ 21%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/network_console_conn.cpp.o
[ 22%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/network_server.cpp.o
[ 22%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/network_stun.cpp.o
[ 22%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/packer.cpp.o
[ 24%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/protocol_ex.cpp.o
[ 24%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/ringbuffer.cpp.o
[ 24%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/serverinfo.cpp.o
[ 25%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/snapshot.cpp.o
[ 25%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/storage.cpp.o
[ 25%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/stun.cpp.o
[ 26%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/teehistorian_ex.cpp.o
[ 26%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/uuid_manager.cpp.o
[ 26%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/video.cpp.o
[ 26%] Building CXX object CMakeFiles/engine-shared.dir/src/engine/shared/websockets.cpp.o
[ 27%] Building CXX object CMakeFiles/engine-shared.dir/src/game/generated/protocolglue.cpp.o
[ 27%] Building CXX object CMakeFiles/engine-shared.dir/src/game/generated/protocol7.cpp.o
[ 27%] Building CXX object CMakeFiles/engine-shared.dir/src/game/generated/protocol.cpp.o
[ 28%] Building CXX object CMakeFiles/engine-shared.dir/src/game/mapitems_ex.cpp.o
[ 28%] Building CXX object CMakeFiles/engine-shared.dir/src/base/hash.cpp.o
[ 28%] Building CXX object CMakeFiles/engine-shared.dir/src/base/hash_bundled.cpp.o
[ 30%] Building CXX object CMakeFiles/engine-shared.dir/src/base/hash_libtomcrypt.cpp.o
[ 30%] Building CXX object CMakeFiles/engine-shared.dir/src/base/log.cpp.o
[ 30%] Building CXX object CMakeFiles/engine-shared.dir/src/base/hash_openssl.cpp.o
[ 31%] Building CXX object CMakeFiles/engine-shared.dir/src/base/system.cpp.o
[ 31%] Building CXX object CMakeFiles/engine-shared.dir/src/base/unicode/confusables.cpp.o
[ 31%] Building CXX object CMakeFiles/engine-shared.dir/src/base/unicode/tolower.cpp.o
[ 31%] Built target engine-shared
[ 31%] Generating src/game/generated/wordlist.h
[ 31%] Generating src/game/generated/client_data7.h
[ 33%] Building CXX object CMakeFiles/config_store.dir/src/tools/config_store.cpp.o
[ 33%] Building CXX object CMakeFiles/dilate.dir/src/tools/dilate.cpp.o
[ 33%] Building CXX object CMakeFiles/crapnet.dir/src/tools/crapnet.cpp.o
[ 33%] Building CXX object CMakeFiles/config_retrieve.dir/src/tools/config_retrieve.cpp.o
[ 33%] Building CXX object CMakeFiles/dummy_map.dir/src/tools/dummy_map.cpp.o
[ 34%] Building CXX object CMakeFiles/map_convert_07.dir/src/tools/map_convert_07.cpp.o
[ 34%] Generating src/game/generated/server_data.cpp
[ 34%] Generating src/game/generated/checksum.cpp
[ 36%] Generating src/game/generated/server_data.h
[ 36%] Generating src/game/generated/client_data.cpp
[ 36%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/antibot.cpp.o
[ 36%] Generating src/game/generated/client_data.h
[ 37%] Generating src/game/generated/client_data7.cpp
[ 37%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/backend_base.cpp.o
[ 37%] Linking CXX executable dilate
[ 37%] Linking CXX executable dummy_map
[ 37%] Linking CXX executable crapnet
[ 38%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/authmanager.cpp.o
[ 38%] Built target dilate
[ 38%] Building CXX object CMakeFiles/map_diff.dir/src/tools/map_diff.cpp.o
[ 38%] Built target dummy_map
[ 38%] Built target crapnet
[ 39%] Linking CXX executable config_retrieve
[ 39%] Building CXX object CMakeFiles/map_extract.dir/src/tools/map_extract.cpp.o
[ 40%] Building CXX object CMakeFiles/map_optimize.dir/src/tools/map_optimize.cpp.o
[ 40%] Built target config_retrieve
[ 42%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/glsl_shader_compiler.cpp.o
[ 42%] Linking CXX executable config_store
[ 42%] Built target config_store
[ 42%] Building CXX object CMakeFiles/map_replace_image.dir/src/tools/map_replace_image.cpp.o
[ 42%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/databases/connection.cpp.o
[ 42%] Linking CXX executable map_convert_07
[ 42%] Built target map_convert_07
[ 42%] Building CXX object CMakeFiles/map_resave.dir/src/tools/map_resave.cpp.o
[ 42%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/null/backend_null.cpp.o
[ 42%] Linking CXX executable map_extract
[ 42%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/databases/connection_pool.cpp.o
[ 42%] Built target map_extract
[ 42%] Building CXX object CMakeFiles/packetgen.dir/src/tools/packetgen.cpp.o
[ 42%] Linking CXX executable map_resave
[ 42%] Linking CXX executable map_optimize
[ 42%] Linking CXX executable packetgen
[ 42%] Built target map_resave
[ 42%] Building CXX object CMakeFiles/stun.dir/src/tools/stun.cpp.o
[ 42%] Built target map_optimize
[ 42%] Built target packetgen
[ 43%] Building CXX object CMakeFiles/unicode_confusables.dir/src/tools/unicode_confusables.cpp.o
[ 43%] Building CXX object CMakeFiles/uuid.dir/src/tools/uuid.cpp.o
[ 44%] Linking CXX executable map_diff
[ 44%] Built target map_diff
[ 45%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/databases/mysql.cpp.o
[ 45%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/opengl/backend_opengl.cpp.o
[ 46%] Linking CXX executable map_replace_image
[ 46%] Built target map_replace_image
[ 48%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/opengl/backend_opengl3.cpp.o
[ 48%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/databases/sqlite.cpp.o
[ 48%] Linking CXX executable stun
[ 48%] Linking CXX executable unicode_confusables
[ 49%] Linking CXX executable uuid
[ 49%] Built target stun
[ 49%] Built target unicode_confusables
[ 49%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/opengl/opengl_sl.cpp.o
[ 49%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/name_ban.cpp.o
[ 49%] Built target uuid
[ 50%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/register.cpp.o
[ 50%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/server.cpp.o
[ 50%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/opengl/opengl_sl_program.cpp.o
[ 50%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/sql_string_helpers.cpp.o
[ 51%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/opengles/backend_opengles.cpp.o
[ 53%] Building CXX object CMakeFiles/DDNet-Server.dir/src/engine/server/upnp.cpp.o
[ 53%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/ddracechat.cpp.o
[ 53%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/ddracecommands.cpp.o
[ 54%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/character.cpp.o
[ 54%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/opengles/backend_opengles3.cpp.o
[ 54%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/opengles/opengles_sl.cpp.o
[ 55%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/opengles/opengles_sl_program.cpp.o
[ 55%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/door.cpp.o
[ 55%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend/vulkan/backend_vulkan.cpp.o
[ 55%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/dragger.cpp.o
[ 55%] Building CXX object CMakeFiles/DDNet.dir/src/engine/client/backend_sdl.cpp.o
/home/lukas/ddnet/src/engine/client/backend/vulkan/backend_vulkan.cpp:41:10: fatal error: vulkan/vulkan.h: No such file or directory
   41 | #include <vulkan/vulkan.h>
      |          ^~~~~~~~~~~~~~~~~
compilation terminated.
make[2]: *** [CMakeFiles/DDNet.dir/build.make:698: CMakeFiles/DDNet.dir/src/engine/client/backend/vulkan/backend_vulkan.cpp.o] Error 1
make[2]: *** Waiting for unfinished jobs....
[ 56%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/dragger_beam.cpp.o
[ 56%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/flag.cpp.o
[ 56%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/gun.cpp.o
[ 57%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/laser.cpp.o
[ 57%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/light.cpp.o
[ 57%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/pickup.cpp.o
[ 59%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/plasma.cpp.o
[ 59%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entities/projectile.cpp.o
[ 59%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/entity.cpp.o
[ 60%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/eventhandler.cpp.o
[ 60%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/gamecontext.cpp.o
[ 60%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/gamecontroller.cpp.o
[ 61%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/gamemodes/DDRace.cpp.o
[ 61%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/gameworld.cpp.o
[ 61%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/player.cpp.o
make[1]: *** [CMakeFiles/Makefile2:292: CMakeFiles/DDNet.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
[ 62%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/save.cpp.o
[ 62%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/score.cpp.o
[ 62%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/scoreworker.cpp.o
[ 63%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/teams.cpp.o
[ 63%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/teehistorian.cpp.o
[ 63%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/server/teeinfo.cpp.o
[ 65%] Building CXX object CMakeFiles/DDNet-Server.dir/src/game/generated/server_data.cpp.o
[ 65%] Linking CXX executable DDNet-Server
[ 65%] Built target DDNet-Server
make: *** [Makefile:156: all] Error 2
```
I saw vulkan in ddnet-libs. Is it only for windows or for all os?
Is it mandatory to install additional libraries to the ddnet-libs for ddnet?

Tried to build the latest master.

After installing vulkan-intel, vulkan-headers and vulkan-icd-loader I could build.

--- a/README.md
+++ b/README.md
@@ -64,7 +64,9 @@ To compile DDNet yourself, execute the following commands in the source root:
     cmake ..
     make -j$(nproc)
 
-Pass the number of threads for compilation to `make -j`. `$(nproc)` in this case returns the number of processing units. DDNet requires additional libraries, that are bundled for the most common platforms (Windows, Mac, Linux, all x86 and x86\_64). The bundled libraries are now in the ddnet-libs submodule.
+Pass the number of threads for compilation to `make -j`. `$(nproc)` in this case returns the number of processing units.
+
+DDNet requires additional libraries, some of which are bundled for the most common platforms (Windows, Mac, Linux, all x86 and x86\_64) for convenience and the official builds. The bundled libraries for official builds are now in the ddnet-libs submodule. Note that when you build and develop locally, you should ideally use your system's package manager to install the dependencies, instead of relying on ddnet-libs submodule, which does not contain all dependencies anyway (e.g. openssl, vulkan). See the previous section for how to get the dependencies. Alternatively see the following build arguments for how to disable some features and their dependencies (`-DVULKAN=OFF` won't require Vulkan for example).
 
 The following is a non-exhaustive list of build arguments that can be passed to the `cmake` command-line tool in order to enable or disable options in build time:
 

